test/integration/targets/aws_ec2_inventory/playbooks/populate_cache.yml

```diff
@@ -0,0 +1,122 @@
+---
+- hosts: 127.0.0.1
+  connection: local
+  gather_facts: no
+  vars:
+    images:
+      us-east-1: ami-4fffc834
+      us-east-2: ami-ea87a78f
```

Comment:
This could be done region-independently with `ec2_ami_facts`. For CentOS I have a canned task: 

```
- ec2_ami_facts:
    # https://wiki.centos.org/Cloud/AWS collected 2018-01-10
    filters:
      architecture: x86_64
      product-code: aw0evgkw8e5c1q413zgy5pjce
      virtualization-type: hvm 
      root-device-type: ebs 
      name: 'CentOS Linux 7*'
```